web/modules/custom/ai_screening_reports/src/Controller/AiScreeningReportsController.php

```diff
@@ -97,6 +99,7 @@ public function projectTrack(Request $request): array|RedirectResponse {
 
       // Create a dataset for each project track.
       foreach ($projectTracks as $projectTrack) {
+        $a = $this->projectTrackHelper->getProjectTrackMaxPossible($projectTrack);
```

Comment:
Is this a leftover from a debugging session?

Reply:
branch maix up. THis has been removed again.